lib/cuvva-error.js

```diff
@@ -15,7 +15,9 @@ function CuvvaError(code, reasons, meta) {
 	}
 
 	CuvvaError.super_(code);
-	Error.captureStackTrace(this, this.constructor);
+
+	if (Error.captureStackTrace)
+		Error.captureStackTrace(this, this.constructor);
```

Comment:
We should move to `Error.stack` instead, this of `captureStackTrace` (which is more V8 specific)

```suggestion
	if (Error.stack)
		this.stack = Error.stack;
```

maybe we can keep the fallback to `captureStackTrace` if necessary (`Error.stack` is more supported but could still fail). `captureStackTrace` doesn't seem to do more than setting the `.stack` variable ourselves (https://nodejs.org/api/errors.html#errors_error_capturestacktrace_targetobject_constructoropt)